scripts/run-tests.sh

```diff
@@ -56,4 +56,6 @@ execDBStatement "source /srv/root/migrations/base.sql"
 
 # Run tests
 echo "Running tests..."
-pytest -vv -s tests/
+coverage run -m pytest -vv -s tests/
+coverage report --show-missing --fail-under=45
+coverage html
```

Comment:
We're currently at 49% coverage. I think we should never *decrease* in coverage -- as we increase our coverage, we should continue to bump this 45%.